docs/pages/brownfield/lifecycle-listeners.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Configuring lifecycle listeners
+sidebar_title: Lifecycle listeners
+description: Learn about the mechanism that allows the Expo Modules API to hook into the lifecycle of your app.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { PlatformTag } from '~/ui/components/Tag/PlatformTag';
+import { DiffBlock } from '~/ui/components/Snippet';
```

Comment:
```suggestion
import { DiffBlock, Terminal } from '~/ui/components/Snippet';
```

Since both of these components come from the same group. This will help resolve the CI errors.